### PR TITLE
Update test cases for sentinel

### PR DESCRIPTION
**Pull request by hwware**
Used the sentinel debug command to reduce the overall time it takes for sentinel test cases to run.
Test time reduced from 3 minutes to 1 minute 40 seconds in local environment.

PR:

https://github.com/redis/redis/pull/9408

### CODE REVIEW

File: tests/sentinel/tests/00-base.tcl [hunk 1]
```diff
@@ -2,6 +2,10 @@
 source "../tests/includes/start-init-tests.tcl"
 source "../tests/includes/init-tests.tcl"
 
+foreach_sentinel_id id {
```
Comment (daniel-house) [inline tests/sentinel/tests/00-base.tcl:5]: The name of this function suggests that it performs some kind of loop, but I can't see that in the body.  In fact, I cannot see any relationship between the function name and what it does.  Perhaps a name like set-default-down-after-to-1000 would be better.

File: tests/sentinel/tests/10-replica-priority.tcl [hunk 2]
```diff
@@ -34,6 +34,9 @@ proc 10_test_number_of_replicas {n_replicas_expected} {
         # ensure sentinels replies with the right number of replicas
         foreach_sentinel_id id {
             # retries 40 x 500ms = 20s as SENTINEL_INFO_PERIOD = 10s
+            S $id sentinel debug info-period 100
+            S $id sentinel debug default-down-after 1000
+            S $id sentinel debug publish-period 100
             set len [llength [S $id SENTINEL REPLICAS mymaster]]
             wait_for_condition 40 500 {
```
Comment (hwware) [inline tests/sentinel/tests/10-replica-priority.tcl]: Here we already decease info-period to 0.1 second, if wait_for_condition parameter could be decreased?

File: tests/sentinel/tests/07-down-conditions.tcl [hunk 3]
```diff
@@ -5,14 +5,23 @@ source "../../../tests/support/cli.tcl"
...
     }
 }
 
+
 proc ensure_master_down {} {
+    S $::alive_sentinel sentinel debug info-period 1000
+    S $::alive_sentinel sentinel debug ping-period 100
+    S $::alive_sentinel sentinel debug ask-period 100
+    S $::alive_sentinel sentinel debug publish-period 100
     wait_for_condition 1000 50 {
```
Comment (hwware) [inline tests/sentinel/tests/07-down-conditions.tcl:25]: same as below comments

File: tests/sentinel/tests/07-down-conditions.tcl [hunk 4]
```diff
@@ -5,14 +5,23 @@ source "../../../tests/support/cli.tcl"
 
 set ::alive_sentinel [expr {$::instances_count/2+2}]
 proc ensure_master_up {} {
+    S $::alive_sentinel sentinel debug info-period 1000
+    S $::alive_sentinel sentinel debug ping-period 100
+    S $::alive_sentinel sentinel debug ask-period 100
+    S $::alive_sentinel sentinel debug publish-period 100
     wait_for_condition 1000 50 {
```
Comment (hwware) [inline tests/sentinel/tests/07-down-conditions.tcl:12]: same as below comments

**Pull request merged**